apps/system/index.html

```diff
@@ -14,6 +14,7 @@
     <script src="shared/js/l10n_date.js"></script>
     <script src="shared/js/idletimer.js"></script>
     <script src="shared/js/dump.js"></script>
+    <script defer src="js/acl_manager.js"></script>
```

Comment:
This would need to change when going to master as we want to avoid boot slowdown. Need to lazyload it using the new loader stuff.